src/sim.ts

```diff
@@ -7,7 +7,7 @@ export class Sim {
 
   constructor() {
     this.vis = new Vis();
-    this.mainPlane = new Plane(1.1, 0.4, 0.1);
+    this.mainPlane = new Plane();
```

Comment:
You might want to define multiple Planes, so I'd recommend moving your input handling and parsing out of the constructor of Plane and into your sim or even main.ts. I think long-term you'd probably want it in main.ts so you can put all the code that deals with handling user inputs in one high-level place.

Reply:
Moving user inputs to main sounds like a good idea!